Server/src/tests/controllers/CategoryController.spec.ts

```diff
@@ -9,7 +9,14 @@ describe('Category Controller', () => {
   let controller: CategoryController;
 
   beforeEach(async () => {
-    await createConnection();
+    try {
+      await createConnection();
+    } catch (error) {
+      // If AlreadyHasActiveConnectionError occurs, return already existent connection
+      if (error.name === "AlreadyHasActiveConnectionError") {
+        return getConnectionManager().get();

```

Comment:
what's the point of returning a getConnectionManager().get() if that is not used, just have an empty catch